### ISSUE TITLE
matCardTitle overridden by h2 styles

### ISSUE THREAD

**Issue by calebegg** [angular/material2#10215]
https://stackblitz.com/edit/angular-material2-issue-wjytux?file=app/app.component.html

I'd like to change my `<mat-card-title>` into `<h2 matCardTitle>`, for screenreader accessibility. But that overrides the styling. I think the styling for matCardTitle needs to be higher specificity.

(I can't use aria-level because it's not widely supported).

**Comment by jelbourn** [comment]
`.mat-typography h2` is indeed overriding the card-title's specificity.

**Comment by jelbourn** [comment]
@crisbeto looking through older issues- it seems like for this one we should probably make sure we have the behavior we want for MDC-based card and let that be the fix. Thoughts?

**Comment by crisbeto** [comment]
Sure, but I think that we'll need a similar fix to #10217 for the MDC card, because it has the same specificity issue.

**Comment by jelbourn** [comment]
Yeah, I meant just fixing it exclusively for mdc card and not worrying about the pre-mdc version.

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
